compiler/config.py

```diff
@@ -127,6 +127,10 @@ def add_common_arguments(parser):
                         help="Run multiple pipelines in parallel if they are safe to run",
                         action="store_true",
                         default=False)
+    parser.add_argument("--parallel_pipelines_limit",
```

Comment:
Let's rename this and parallel_pipelines to dependency_untangling now that you are at it :)

Reply:
Planning on keeping it like this for now since the benchmarking scripts use parallel_pipelines. We can change it in a different pull request.